benchmarks/deltablue_perms/comp.php

```diff
@@ -0,0 +1,362 @@
+<?php{
+// Copyright 2008 the V8 project authors. All rights reserved.
+// Copyright 1996 John Maloney and Mario Wolczko.
+
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+
+// Ported to PHP from Google's Octane v2.0 benchmarking suite for JavaScript.
+
+// This implementation of the DeltaBlue benchmark is derived
+// from the Smalltalk implementation by John Maloney and Mario
+// Wolczko. Some parts have been translated directly, whereas
+// others have been modified more aggresively to make it feel
+// more like a JavaScript program.
+
+// Global variable holding the current planner.
+$planner = null;
+
+compile_py_func_global("def alert(s):\n    raise Exception(\"Alert called with argument %s\" % s)");
+
+class OrderedCollection {
+  private $elms;
+  
```

Comment:
Because this is based upon a file eco generated. We should leave them in for diffing purposes.